### PR TITLE
Fixes issue #2230

### PR DESCRIPTION
**Pull request by DarkPrince304**
This will fix the issue zaproxy/zaproxy#2230 in zaproxy.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/pscanrulesBeta/InsecureAuthenticationScan.java [hunk 1]
```diff
@@ -240,11 +240,12 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
...
 					alert.setDetail(getDescription(),
 							msg.getRequestHeader().getURI().toString(),
-							"", HttpHeader.WWW_AUTHENTICATE + ": " + auth,
+							"",
+							"",
 							"",
 							getSolution(),
 							getReference(),
-							"",	// No Evidence
+							HttpHeader.WWW_AUTHENTICATE + ": " + auth,	// No Evidence
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesBeta/InsecureAuthenticationScan.java]: The comment "// No Evidence" should be removed.

**Comment by thc202** [comment]
It would be great if the commit message contained a short description of what (and why) was fixed.

The reference to the issue should be prefixed with "zaproxy/zaproxy#" otherwise it's referencing an issue in zap-extensions repo.

**Comment by thc202** [comment]
BTW, might worth rebase the branch to bring it up-to-date with latest changes.

**Comment by thc202** [comment]
Looks good to me.

**Comment by psiinon** [comment]
Looks good to me as well :)
Thanks for contributing!

**Pull request merged**